components/core/src/package/install.rs

```diff
@@ -22,9 +22,9 @@ use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
 pub use types::package_install::*;
+use super::{Identifiable, MetaFile, PackageIdent, Target, PackageTarget};
```

Comment:
I think it's more appropriate to use `super` and `self` because it makes the module more self contained. It can be picked up and moved anywhere else in the codebase without worrying about where on disk it resides and which modules surround it